### ISSUE TITLE
Set up github actions for checking whether the code compiles or not

### ISSUE THREAD

**Issue by vishalvibes** [houseofgeeks/CP-Dictionary#181]
Set up github actions for checking whether the code compiles or not. Please put your views about implementing it and how can we do it for different languages!

**Comment by anktrj** [comment]
That's a very good idea!

**Comment by prskid1000** [comment]
i think we have to create it from scratch.
We can start it as another project of our organization.It is bound to be most popular one as this type of action is not available in marketplace.

**Comment by sanskarseth** [comment]
Yes having a separate project is a better option than only implementing In this.

**Comment by EmperorYP7** [comment]
> We can start it as another project of our organization.It is bound to be most popular one as this type of action is not available in marketplace.

That sounds great! You up for this @tusharjain0022?

**Comment by tusharjain0022** [comment]
> > We can start it as another project of our organization.It is bound to be most popular one as this type of action is not available in marketplace.
> 
> That sounds great! You up for this @tusharjain0022?

I can try this 🙌
